pkg/karmadactl/cmdinit/cert/cert_test.go

```diff
@@ -101,7 +124,7 @@ func TestGenCerts(_ *testing.T) {
 	apiserverCertCfg := NewCertConfig("karmada-apiserver", []string{""}, karmadaAltNames, &notAfter)
 	frontProxyClientCertCfg := NewCertConfig("front-proxy-client", []string{}, certutil.AltNames{}, &notAfter)
 
-	if err := GenCerts(TestCertsTmp, etcdServerCertConfig, etcdClientCertCfg, karmadaCertCfg, apiserverCertCfg, frontProxyClientCertCfg); err != nil {
+	if err := GenCerts(TestCertsTmp, TestCaCertPath, TestCaKeyPath, etcdServerCertConfig, etcdClientCertCfg, karmadaCertCfg, apiserverCertCfg, frontProxyClientCertCfg); err != nil {
```

Comment:
May be two test function `GenCerts(TestCertsTmp, TestCaCertPath, TestCaKeyPath, ...)` and `GenCerts(TestCertsTmp, "", "", ..)` can exist simultaneously

Reply:
ok

Reply:
maybe you even don't need to mock ca cert and key, since after you call `GenCerts(TestCertsTmp, "", "", ..)`, it will generate ca cert and key at `TestCertsTmp`.

Reply:
My fault, since you have changed, let's just keep the current code~